src/modlunky2/levels/tile_codes.py

```diff
@@ -288,7 +393,7 @@ def parse(cls, line: str) -> "TileCode":
             raise ValueError("Directive missing name.")
 
         obj = cls(name, value, comment)
-        obj.validate()
+        # obj.validate()
```

Comment:
I think we'd still want to validate values so just comment out `validate_name` in the `validate` method and put this back